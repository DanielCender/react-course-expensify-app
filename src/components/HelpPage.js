import React from 'react';
import Header from './Header';

export default () => (
	<div>
		<Header />
		<br />
		This is my help page component.
	</div>
);
