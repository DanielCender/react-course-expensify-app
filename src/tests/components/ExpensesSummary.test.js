import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly render an expense summary for 3 expenses', () => {
	const wrapper = shallow(
		<ExpensesSummary expenseCount={3} expensesTotal={300} />,
	);
	expect(wrapper).toMatchSnapshot();
});

test('should correctly render summary for 2 expenses', () => {
	const wrapper = shallow(
		<ExpensesSummary expenseCount={2} expensesTotal={500} />,
	);
	expect(wrapper).toMatchSnapshot();
});
