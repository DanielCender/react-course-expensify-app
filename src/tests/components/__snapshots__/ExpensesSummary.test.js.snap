// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render an expense summary for 3 expenses 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totalling 
    $3.00
  </h1>
</div>
`;

exports[`should correctly render summary for 2 expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling 
    $5.00
  </h1>
</div>
`;
